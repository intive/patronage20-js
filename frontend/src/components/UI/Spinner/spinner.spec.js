/* globals describe, test, expect */

import React from 'react'
import TestRenderer from 'react-test-renderer'
import Spinner from './index'

describe('<Spinner />', () => {
  test('renders Spinner component', () => {
    const root = TestRenderer.create(<Spinner />)
    expect(root.toJSON()).toMatchSnapshot()
  })
})
