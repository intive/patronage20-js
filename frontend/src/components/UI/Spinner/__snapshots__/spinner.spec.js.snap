// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> renders Spinner component 1`] = `
<div
  className="MuiBox-root MuiBox-root-3 makeStyles-root-1"
>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 70,
        "width": 70,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
